modules/check_in/app/sidekiq/check_in/travel_claim_submission_worker.rb

```diff
@@ -57,8 +61,6 @@ def handle_response(opts = {})
                                        [Constants::OH_STATSD_BTSSS_SUCCESS, Constants::OH_SUCCESS_TEMPLATE_ID]
                                      when TravelClaim::Response::CODE_CLAIM_EXISTS
                                        [Constants::OH_STATSD_BTSSS_DUPLICATE, Constants::OH_DUPLICATE_TEMPLATE_ID]
-                                     when TravelClaim::Response::CODE_BTSSS_TIMEOUT
-                                       [Constants::OH_STATSD_BTSSS_TIMEOUT, Constants::OH_TIMEOUT_TEMPLATE_ID]
```

Comment:
Could you please check?  By removing this case statement, we would miss sending out a "`taking longer than usual`" message to the veteran until `check_in_experience_check_claim_status_on_timeout_enabled` is turned on.

Reply:
yes, you're right. will fix.

Reply:
addressed in a516421462a4debd2bd88d08c0bbbf5cfe5d5855